src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCode.cs

```diff
@@ -23,97 +22,89 @@ namespace System.Text.RegularExpressions
+        internal const int Eol = 15;                //                          $
+        internal const int Boundary = 16;           //                          \b
+        internal const int Nonboundary = 17;        //                          \B
+        internal const int Beginning = 18;          //                          \A
+        internal const int Start = 19;              //                          \G
+        internal const int EndZ = 20;               //                          \Z
+        internal const int End = 21;                //                          \Z
+
+        internal const int Nothing = 22;            //                          Reject!
+
+        // Primitive control structures
+
+        internal const int Lazybranch = 23;         // back     jump            straight first
+        internal const int Branchmark = 24;         // back     jump            branch first for loop
+        internal const int Lazybranchmark = 25;     // back     jump            straight first for loop
+        internal const int Nullcount = 26;          // back     val             set counter, null mark
+        internal const int Setcount = 27;           // back     val             set counter, make mark
+        internal const int Branchcount = 28;        // back     jump,limit      branch++ if zero<=c<limit
+        internal const int Lazybranchcount = 29;    // back     jump,limit      same, but straight first
+        internal const int Nullmark = 30;           // back                     save position
+        internal const int Setmark = 31;            // back                     save position
+        internal const int Capturemark = 32;        // back     group           define group
+        internal const int Getmark = 33;            // back                     recall position
+        internal const int Setjump = 34;            // back                     save backtrack state
+        internal const int Backjump = 35;           //                          zap back to saved state
+        internal const int Forejump = 36;           //                          zap backtracking state
+        internal const int Testref = 37;            //                          backtrack if ref undefined
+        internal const int Goto = 38;               //          jump            just go
+
+        internal const int Prune = 39;              //                          prune it baby
+        internal const int Stop = 40;               //                          done!
+
+        internal const int ECMABoundary = 41;       //                          \b
+        internal const int NonECMABoundary = 42;    //                          \B
+
+        // Modifiers for alternate modes
         internal const int Mask = 63;   // Mask to get unmodified ordinary operator
-        internal const int Rtl = 64;   // bit to indicate that we're reverse scanning.
+        internal const int Rtl = 64;    // bit to indicate that we're reverse scanning.
```

Comment:
If you want even more commits, it could be interesting to see if renaming these modifiers yields more readable code where they are used.  I haven't actually looked into it too much, but it might be something to try if you are bored :-).